src/identity.d.ts

```diff
@@ -353,6 +353,76 @@ export type LoginOptions = {
      * in place of default teaser
      */
     teaser?: string;
+    /**
+     * - Specifies the allowable elapsed time in seconds since
+     * the last time the End-User was actively authenticated. If last authentication time is more
+     * than maxAge seconds in the past, re-authentication will be required. See the OpenID Connect
+     * spec section 3.1.2.1 for more information
+     */
+    maxAge?: number | string;
+    /**
+     * - Optional parameter to overwrite client locale setting.
+     * New flows supports nb_NO, fi_FI, sv_SE, en_US
+     */
+    locale?: string;
+    /**
+     * - display username and password on one screen
+     */
+    oneStepLogin?: boolean;
+    /**
+     * - String that specifies whether the Authorization Server prompts the
+     * End-User for reauthentication or confirm account screen. Supported values: `select_account` or `login`
+     */
+    prompt?: string;
+};
+export type SimplifiedLoginWidgetLoginOptions = {
+    /**
+     * - An opaque value used by the client to maintain state between
+     * the request and callback. It's also recommended to prevent CSRF {@link https://tools.ietf.org/html/rfc6749#section-10.12}
+     */
+    state: string | Promise<string> | (() => string);
```

Comment:
I think state could be rather a function returning promise instead of just promise. 
```suggestion
    state: string | (() => Promise<string>) | (() => string);
```

Reply:
A function returning a promise wouldn't work with the code tho, right? 
It expects to do `if(typeof state === "function") state = await state()` and it expects that to be a string, not a Promise

Reply:
Scratch that, I'm wrong, will fix!